Cangjie5.txt

```diff
@@ -103968,6 +103967,7 @@
 烲	xqlf
 𰓐	xqlk
 𫑢	xqll
+𪮋	xqll
```

Comment:
I am not sure how should we deal with duplicate mappings which begin with `x`, are they supposed to be unique?

Reply:
Yes, we keep only one and remove others.

Reply:
Okay in this case I will remove `xqll` for 𪮋.